deepchecks/base/dataset.py

```diff
@@ -184,9 +192,9 @@ def from_numpy(
             *args: (np.ndarray):
                 expecting it to contain two numpy arrays (or at least one),
```

Comment:
This phrasing is not clear (I know its not in the CR but good opportunity to improve it :) )

Reply:
meant the sentence "expecting it to contain two numpy arrays (or at least one), first with features, second with labels."
What does it mean (or at least one), and what is the behavior for each scenario... (one/two)